docs/content/onedrive.md

```diff
@@ -130,8 +130,8 @@ Client ID and Key by following the steps below:
 1. Open https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade and then click `New registration`.
 2. Enter a name for your app, choose account type `Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)`, select `Web` in `Redirect URI`, then type (do not copy and paste) `http://localhost:53682/` and click Register. Copy and keep the `Application (client) ID` under the app name for later use.
 3. Under `manage` select `Certificates & secrets`, click `New client secret`. Copy and keep that secret for later use.
-4. Under `manage` select `API permissions`, click `Add a permission` and select `Microsoft Graph` then select `delegated permissions`.
```

Comment:
Thanks for testing @Cnly 

As far as I can see @Cnly suggested to make updated items 4,5 conditional and provide a description when delegated or administered way should take place.  Let me try to satisfy the suggestion:

---------------------------SNIP-------------------------
3. Under `manage` select `Certificates & secrets`, click `New client secret`. Copy and keep that secret for later use.

The next steps depend on whether you want to use your Client ID personally or within a OneDrive organization.

In case of OneDrive Personal:
- Under `manage` select `API permissions`, click `Add a permission` and select `Microsoft Graph` then select `delegated permissions`.
- Search and select the following permissions: `Files.Read`, `Files.ReadWrite`, `Files.Read.All`, `Files.ReadWrite.All`, `offline_access`, `User.Read`. Once selected click `Add permissions` at the bottom.

If you are going to use your new Client ID within an organization, the required permissions are slightly different. First of all, if you are **not member**, you should contact its administrator and ask them to manually grant the permissions. If you **are member**:
- Under `manage` select `API permissions`, click `Add a permission` and select `Microsoft Graph` then select `Application permissions`.
- Search and select the following permissions: `Files.Read.All`, `Files.ReadWrite.All`, `User.Read.All`. Once selected, click `Add permissions` at the bottom.

Now the application is complete.
------------------------SNIP----------------------------

I am copy pasting intentionally. Instructions with more than one `if` will just confuse users.

@Cnly Please review

Reply:
Thanks for that :D One point that needs clarification: You need to finish the creation of the app (i.e. including configuring the permissions) before an admin can allow the use of that outside app. Alternatively, outside applications can choose to use the old delegated permissions. Do you think this alternative way should be mentioned?